src/prefs/EffectsPrefs.cpp

```diff
@@ -124,6 +127,15 @@ void EffectsPrefs::PopulateOrExchange(ShuttleGui & S)
    }
    S.EndStatic();
 #endif
+
+   S.AddButton(XO("Open Plugin Manager"), wxALIGN_LEFT)->Bind(wxEVT_BUTTON, [this](auto) {
+      //Adding dependency on PluginRegistrationDialog, not good. Alternatively
```

Comment:
A wise comment.  I like it that you are aware of dependencies you add.

But I checked that this does not add a dependency cycle.

True, it may still be an undesirable dependency, but we can figure out some trick to break it if we need that later.